### ISSUE TITLE
Blocked but not logged

### ISSUE THREAD

**Issue by zu1k** [chaitin/safeline#6]
`http://192.168.226.4:38989/?a=1%9c%27union%20select%20*%20from%20user`

Blocked but not logged, caused by invalid UTF-8 char `0x9c`.

```
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] ERROR:  invalid byte sequence for encoding "UTF8": 0x9c
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] CONTEXT:  unnamed portal parameter $7
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] STATEMENT:  INSERT INTO "mgt_detect_log_basic" ("event_id","site_uuid","src_ip","socket_ip","protocol","host","url_path","dst_port","country","province","city","attack_type","risk_level","action","rule_id","timestamp") VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$13,$14,$15,$16) RETURNING "id"
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] ERROR:  invalid byte sequence for encoding "UTF8": 0x9c
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] CONTEXT:  unnamed portal parameter $5
safeline-postgres  | 2023-04-14 09:17:02.375 UTC [123] STATEMENT:  INSERT INTO "mgt_detect_log_detail" ("event_id","src_port","dst_ip","method","query_string","status_code","req_header","req_body","rsp_header","rsp_body","payload","location","decode_path") VALUES ($1,$2,$3,$4,$5,$6,$7,$8,$9,$10,$11,$12,$13) RETURNING "id"
```

**Comment by zclaiqcc** [comment]
ok, we will fix it in the later version

**Comment by dundunHa** [comment]
fix it in the v1.5.0
